app/javascript/app/components/data-explorer-content/data-explorer-content.js

```diff
@@ -165,23 +209,17 @@ class DataExplorerContentContainer extends PureComponent {
       section,
       filterName
     ).map(k => ({ name: `${section}-${k}`, value: undefined }));
-
+    let parsedValue = value;
+    if (multiple) parsedValue = this.parsedMultipleValues(filterName, value);
+    parsedValue = parsedValue === '' ? undefined : parsedValue;
```

Comment:
maybe use shortcut like `parsedValue = parsedValue || undefined` ?

Reply:
Actually, it wasn't needed anymore. Thank you for spotting it